src/mbgl/plugins/plugin_layer.cpp

```diff
@@ -0,0 +1,110 @@
+//
+//  plugin_layer.cpp
+//  App
+//
+//  Created by Malcolm Toon on 4/24/25.
+//
+
+#include <mbgl/plugins/plugin_layer.hpp>
```

Comment:
Nit: can we use

```
src/mbgl/plugin
```

instead of 

```
src/mbgl/plugins
```

since the other directories are also singular?